ci/test_e2e

```diff
@@ -1,20 +1,33 @@
-#!/bin/bash -e
+#!/bin/bash -ex
+
+#
+# Remote Integration Test Runner
+#
+# This runs all features - including those dependent on
```

Comment:
This comment ("all features") will need to change if `test_e2e` is modified to just run the E2E tests that depend upon external PCF/Tanzu access.